include/ua_plugin_nodestore.h

```diff
@@ -483,6 +483,10 @@ UA_StatusCode UA_EXPORT
 UA_Node_setAttributes(UA_Node *node, const void *attributes,
                       const UA_DataType *attributeType);
 
+/* Copy the references from one array to another */
+UA_StatusCode UA_EXPORT
+UA_NodeReferenceKind_copy(const UA_NodeReferenceKind *src, size_t srcSize, UA_NodeReferenceKind **dst);
```

Comment:
We do pretty much the same in UA_Node_copy.
Can we implement once and reuse?

Reply:
The method name is misleading.
It sounds as if we make a copy of one NodeReferenceKind structure.
But we deep-copy an array of NodeReferenceKind structures.